wagtail/admin/views/pages/edit.py

```diff
@@ -868,6 +868,7 @@ def get_side_panels(self):
                 scheduled_object=self.scheduled_page,
                 locale=self.locale,
                 translations=self.translations,
+                parent_page=self.page.get_parent(),
```

Comment:
My only thought is that we probably could get away without adding this, but I can see where it may be expected to be consistent so let's leave it.